src/azure-cli/azure/cli/command_modules/appconfig/tests/latest/recordings/test_azconfig_mgmt.yaml

```diff
@@ -1,986 +0,0 @@
-interactions:
```

Comment:
Why was this file deleted?

Reply:
This is actually something wrong with merging of branches. I will look deeper into it.

Reply:
Okay, this is a combination of merge conflict and some changes in Azure SDK.
The Azure SDK is not generating this file along with a file for another AAD test anymore.
Seems like there is a bug in Azure SDK or a change in behavior.

We should clarify with the SDK team.

@zhoxing-ms any idea why this might be happening?

Reply:
> This is actually something wrong with merging of branches. I will look deeper into it.
Okay, this is a combination of merge conflict and some changes in Azure SDK.

May I ask which branch do you mean has the conflict? At present, this file is not deleted in the remote dev branch.

Reply:
This may be due to some exceptions encountered when executing the test in live mode, it caused the test process to exit early, resulting in the deletion of the original yaml file but it hasn't had time to re-record the new yaml file
Could you sync the latest code from the remote dev branch and then re-record the test in live mode?

Reply:
@zhoxing-ms, Yes, this file is present in the remote dev branch, I meant I resolved conflicts incorrectly which resulted in committing the deleted file.

I tried your suggestion.
1. Pulled the latest changes from remote dev.
2. Rebased my commits over it.
3. Executed all the tests with `--live`

Following are my observations - 
1. The tests are successful in both `--live` and without `--live`.
2. recording files are not generated for a couple of tests, but are generated for other tests.

Do you know what more steps I could possibly take to get to the root of this issue?

Reply:
> recording files are not generated for a couple of tests, but are generated for other tests.

Which tests do not generate recording files, and are they marked as `@live_only`?

Reply:
Hi @zhoxing-ms, no, they are not marked as `@live_only`.

Reply:
Could you tell me which tests do not generate recording files?

Reply:
In the latest run, the only test not generating file is this one -
[test_appconfig_commands.py#L26-L160](https://github.com/pratiksanglikar/azure-cli/blob/prsangli/soft-delete/src/azure-cli/azure/cli/command_modules/appconfig/tests/latest/test_appconfig_commands.py#L26-L160
) and it was run successfully - [test output](https://dev.azure.com/azure-sdk/public/_build/results?buildId=1411036&view=logs&j=15e5eb7c-cf66-5b91-00a2-db9b11b5e0f9&t=845fa2ee-a602-59f5-4e4c-07ee6d0283b9&l=389)
Seems like the issue with other tests has been resolved on it's own.

Reply:
I fixed the test based on @cxznmhdcxz comment. Now it is working.